pkgs/top-level/all-packages.nix

```diff
@@ -2726,6 +2726,8 @@ with pkgs;
 
   goku = callPackage ../os-specific/darwin/goku { };
 
+  grb = callPackage ../applications/misc/grb { };
```

Comment:
Split the commit TODO:

[ ] The new package (including this `grb = callPackage ...` line and the new file) should be in a separate commit.
[ ] Each package you have updated should be in its own separate commit.

Otherwise the code is OK.

Reply:
I did this + updated my maintainer-list.nix info in another separate commit